ai-machine-learning/thoth-thamos/setup.md

```diff
@@ -0,0 +1,29 @@
+Before start using the Thamos let's setup the environment.
+
+This image comes with Python2 so we are going to install Python3 -
+
+``yum install python3``{{execute}}
```

Comment:
Is there a need to install `python3-pip`?

Reply:
It seems to work fine without doing that.

Reply:
OK, thanks for checking it? :+1: